server/dataGeneration/command.js

```diff
@@ -2,22 +2,16 @@ const fs = require('fs');
 const path = require('path');
 
 module.exports = function (j) {
```

Comment:
Interested to know what j is representing here. Maybe you could specify that as the param name instead.

Reply:
Yes, I should be more specific. The data-generations script has to split the total query into several files since the the string it is creating with every iteration would become too big and exceed the available amount of memory. So to address that, I just used the number of the iteration on each file (e.g. reviewsBody0.sql, reviewsBody1.sql, etc) and the final number for the iterations is used in this command to generate a `cat` script with the correct amount of files. 

Sorry I don't think I am explaining it very well, I can show you on zoom sometime if you ever want to see it in action.